### ISSUE TITLE
Does KC..Default. correspond to questions or higher level constructs?

### ISSUE THREAD

**Issue by erkaner** [Optimal-Learning-Lab/LKT#16]
I'd like to clarify my understanding of the term "KC." From what I've gathered, KC seems to stand for "Knowledge Component" and appears to have different interpretations. Initially, I thought KC referred to the building blocks or elements that students are assessed on, such as specific skills or concepts. For instance, a learning objective like "students will be able to multiply multi-digit whole numbers" or a larger topic like "multiplication of multi-digit numbers" could be considered as KCs.

However, based on the examples provided in the Vignettes as well as the LKT Webinars I watched, it appears that KCs are actually the individual assessment questions themselves, not just the broader components or topics. 

Should KC strictly refer to the questions in the assessment in LKT, or do we have the flexibility to define KC based on our specific educational context? I'm seeking clarification to ensure I have a solid understanding of the term and its implications.

**Comment by imrryr** [comment]
Yes, a KC could be either, though the "skill" interpretation is typically more useful in most contexts rather than the "question" interpretation. KC assignments of items to skills or just as items should be chosen in a way to best represent the relationships in the data. Often experts codes KCs, but there are also automatic methods to derive the relationships... (often the results from these are a bit hard to interpret).
